tsconfig.common.json

```diff
@@ -0,0 +1,26 @@
+{
+  "extends": "@tsconfig/node20/tsconfig.json",
+  "exclude": ["node_modules"],
+  "include": ["./jest.config.ts", "./src/**/*.ts", "./__mocks__/**/*.ts"],
+  "compilerOptions": {
+    "declaration": true,
+    "declarationMap": true,
+    "emitDecoratorMetadata": true,
+    "esModuleInterop": true,
+    "experimentalDecorators": true,
+    "importHelpers": true,
+    "module": "node16",
+    "moduleResolution": "node16",
+    "noEmit": true,
+    "resolveJsonModule": true,
+    "rootDir": ".",
+    "sourceMap": true,
+    "strict": true,
+    "strictNullChecks": true,
+    "target": "es2020",
```

Comment:
I'd argue that with this release we should target nodejs 18 at least (since node16 is no longer supported) so should go for an `ES2022` target.

Reply:
Indeed, I changed it.